operations/finance.md

```diff
@@ -693,7 +696,8 @@ Returns all credit cards with the specified ids.
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `CreditCardIds` | array of string | required | Unique identifier of the [Credit card](finance.md#credit-card). |
+| `CreditCardIds` | array of string | optional | Unique identifiers of the [Credit card](finance.md#credit-card). |
+| `CustomerIds` | array of string | required | Unique identifiers of the [Customer](customers.md#customer)s. |
```

Comment:
Is it really required ? Just one of those 2 are required right ?

Reply:
You're right, although both are optional, at least one of them must be specified.